lib/pbench/test/unit/server/query_apis/test_month_indices.py

```diff
@@ -27,7 +26,7 @@ def get_helper(expected_status, server_config, **kwargs):
         host = server_config.get("elasticsearch", "host")
         port = server_config.get("elasticsearch", "port")
         es_url = f"http://{host}:{port}"
-        requests_mock.get(re.compile(f"{es_url}"), **kwargs)
+        requests_mock.get(es_url + "/_aliases", **kwargs)
         response = client.get(f"{server_config.rest_uri}/controllers/months")
         assert requests_mock.last_request.url == (es_url + "/_aliases")
```

Comment:
Did you want to do here what you did in `lib/pbench/test/unit/server/query_apis/conftest.py` and put `/_aliases` into `es_url`?

Reply:
I noticed that right after I pushed the commit. Dang.